data_structures/linked_list/rotate_linked_list_by_k.py

```diff
@@ -7,12 +7,15 @@ def __init__(self, data: int) -> None:
         self.next = None
 
 
-def print_linked_list(head: Node) -> None:
+def print_linked_list(head: Node | None) -> None:
     """
     Print the entire linked list iteratively.
 
+    This function prints the elements of a linked list iteratively, separated by '->'.
+
     Parameters:
-        head (Node): The head of the linked list to be printed.
+        head (Node | None): The head of the linked list to be printed,\
```

Comment:
PEP8.  Do not use backslash line termination in Python

```suggestion
        head (Node | None): The head of the linked list to be printed,
```